lineage/data/test.csv

```diff
@@ -0,0 +1,114 @@
+frame,trackId,lineageId,parentTrackId,Diameter_0
+0,1,2,0,10

```

Comment:
Why not test with the actual data?

Reply:
At first, I did, but thought since I have to use a part of the code to put the cells from each lineage together, it might be better to make a synthetic one. If you don't agree, I will edit it and use the data itself.

Reply:
I think it'd be better to import and test of the real data file. After that this should be ready to merge.

Reply:
done.